### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by bebatut**
- Add Taj to people file
- Add title to team members
- Add team members to team page
- Fix title display in people box

### CODE REVIEW

File: _data/community.yaml [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+  - bebatut
+  - emmyft
+  - malvikasharan
+  - yochannah
+  - pazbc
+  - tajuddeen1
+  - pherterich
+  - msundukova
+  - npdebs
+  - jilaga
```
Comment (yochannah) [inline _data/community.yaml:12]:
```suggestion
  - jilaga
  - flavioazevedo
  - bethaniley
  - Camille-Gonz-VA
  - iramosp
```

File: _data/people.yaml [hunk 2]
```diff
@@ -1415,6 +1416,10 @@ camila-gómez:
   first-name: Camila
   last-name: Gómez
   orcid: 0000-0001-7642-6049
+camille-gonz-va:
+  first-name: Camille
+  last-name: Gonzalez
+  title: Executive assistant (contract)
```
Comment (bebatut) [inline _data/people.yaml:1422]:
```suggestion
  title: Executive Assistant (contract)
```

File: _data/people.yaml [hunk 3]
```diff
@@ -5589,6 +5603,7 @@ pazbc:
   pronouns: she/her
   twitter: PazByC
   website: https://openlifesci.org/posts/2022/08/10/welcome-paz/
+  title: Community researcher and Programme coordinator (contract)
```
Comment (bebatut) [inline _data/people.yaml:5606]:
```suggestion
  title: Community Researcher & Programme Coordinator (contract)
```

File: _data/people.yaml [hunk 4]
```diff
@@ -5673,6 +5688,7 @@ pherterich:
   orcid: 0000-0002-4542-9906
   pronouns: she/her
   twitter: pherterich
+  title: Finance coordinator and Resident fellow
```
Comment (bebatut) [inline _data/people.yaml:5691]:
```suggestion
  title: Finance Coordinator & Resident Fellow
```

File: _data/people.yaml [hunk 5]
```diff
@@ -6701,6 +6717,11 @@ tai-rocha:
   last-name: Rocha
   orcid: 0000-0001-6874-2447
   twitter: Tai_Rocha_
+tajuddeen1:
+  country: Nigeria
+  first-name: Tajuddeen
+  last-name: Gwadabe
+  title: Programme manager, Catalyst (contract)
```
Comment (bebatut) [inline _data/people.yaml:6724]:
```suggestion
  title: Programme Manager & Catalyst (contract)
```

**Comment by pherterich** [comment]
I might be "Fellowship and finance manager" from Monday on and have a feeling that I might break everything if I try to do this in a PR myself 😬

File: _data/people.yaml [hunk 6]
```diff
@@ -5673,6 +5688,7 @@ pherterich:
   orcid: 0000-0002-4542-9906
   pronouns: she/her
   twitter: pherterich
+  title: Finance Coordinator & Resident Fellow
```
Comment (yochannah) [inline _data/people.yaml:5691]:
```suggestion
  title: Fellowship and Finance Manager
```

**Review by yochannah — APPROVED** [review]

**Comment by yochannah** [comment]
Thanks @bebatut !!!

**Pull request merged**